shopinvader/wizards/shopinvader_category_binding_wizard.py

```diff
@@ -34,18 +34,15 @@ class ShopinvaderCategoryBindingWizard(models.TransientModel):
 
     @api.model
     def default_get(self, fields_list):
-        result = super(ShopinvaderCategoryBindingWizard, self).default_get(
+        res = super(ShopinvaderCategoryBindingWizard, self).default_get(
             fields_list
         )
-        backend_id = self.env.context.get("active_id")
+        backend_id = self.backend_id.browse(
```

Comment:
`s/backend_id/backend` as it is a record and not and ID

Reply:
As far as I know, mixing ids and records within *_id is everywhere in odoo / OCA.
Is it a new convention to stop doing it ?

Reply:
For me, the OCA convention is: we try to do it better than before.
Every time I see a variable named `_id` when it's a record I :tableflip: :smile: 
So when possible, let's try to keep it clean. 
In any case, I think that it's just a bad inheritance from the days before new api came, whereas you had IDs all around and they simply did not change var names.